BridgeSDK/BridgeAPI/SBBAuthManager.m

```diff
@@ -363,6 +363,34 @@ - (NSURLSessionDataTask *)signOutWithCompletion:(SBBNetworkManagerCompletionBloc
     }];
 }
 
+- (NSString *)savedUsername
+{
+    NSString *username = nil;
+    if (_authDelegate) {
+        if ([_authDelegate respondsToSelector:@selector(usernameForAuthManager:)]) {
```

Comment:
why the double nesting? This means that it will only fallback to the usernameFromKeychain if the authDelegate is nil, rather than if the authDelegate does not implement the protocol method?

````
if ([_authDelegate respondsToSelector:@selector(usernameForAuthManager:)]) { 
    return [_authDelegate usernameForAuthManager:self];
} else {
    return [self usernameFromKeychain];
}
````